sdk/contentsafety/azure-ai-contentsafety/samples/sample_analyze_image.py

```diff
@@ -0,0 +1,38 @@
+import os
+from azure.ai.contentsafety import ContentSafetyClient
+from azure.core.credentials import AzureKeyCredential
+from azure.ai.contentsafety.models import AnalyzeImageOptions, ImageData
+
+def analyze_image():
+    key = os.environ["CONTENT_SAFETY_KEY"]
+    endpoint = os.environ["CONTENT_SAFETY_ENDPOINT"]
+    image_path = os.path.join("sample_data", "image.jpg")
```

Comment:
FYI I tried to run this sample locally and the file couldnt be found. You could try using the absolute path. Here is an example: https://github.com/Azure/azure-sdk-for-python/blob/335490752d62e77568651df9b01f5cb19b27fd2a/sdk/formrecognizer/azure-ai-formrecognizer/samples/v3.2/sample_analyze_layout.py#L38

Reply:
Fxied.